sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/UcfgInstructionFactory.cs

```diff
@@ -478,6 +473,30 @@ private IEnumerable<Instruction> ProcessMemberAccessExpression(MemberAccessExpre
             return instructions;
         }
 
+        private IEnumerable<Instruction> ProcessOperator(SyntaxNode syntaxNode, UcfgExpression leftExpression,
+            UcfgExpression rightExpression)
+        {
+            var operatorMethodSymbol = GetSymbol<IMethodSymbol>(syntaxNode);
+            if (operatorMethodSymbol == null)
+            {
+                expressionService.Associate(syntaxNode, expressionService.CreateConstant());
+                return NoInstructions;
+            }
+
+            var isStringConcat = operatorMethodSymbol.MethodKind == MethodKind.BuiltinOperator &&
+                operatorMethodSymbol.Parameters.Length == 2 &&
+                operatorMethodSymbol.Name == "op_Addition" &&
+                operatorMethodSymbol.ContainingType.Is(KnownType.System_String);
```

Comment:
?Test to check we don't fail for other binary and unary operators: ==, !=, -=, !